test/fuzzy_object_store/README.md

```diff
@@ -0,0 +1,77 @@
+# The Fuzz Framework project
+
+This project is an attempt to put together all the small fuzzers we have already scattered around the code.
+There are two goals:
+    1. To be able to run all the fuzzers, collect crashes reports and fix possible bugs that the fuzzer might find.
+    2. To be able to replace libfuzzer with google fuzz test (https://github.com/google/fuzztest) at some point.
+
+AFL++ support is not dropped yet, but since we want to integrate things inside evergreen and follow the same approach we implement for address/thread sanitazer we prefer to use libfuzzer and clang.
+## Prerequisites
+
+In case you want to use AFL++, then you should install the latest version of the American Fuzzy Lop ++ (AFL++).
+Please use this quick guide: https://aflplus.plus/building/ it requires llvm >= 9.0.
+
+For using libfuzzer, the only pre-requisite is having a recent version of clang.
+## Running
+Note REALM_MAX_BPNODE_SIZE is the max number of nodes contained in the b+tree. It determines the depth of the tree and its fanout. \
+This number should be random generated.
```

Comment:
How random (range)?
We build core with one number, so in theory that should be used. But a low number will provoke testing more of the code with less data.

Reply:
Are you suggesting having this set as an optional parameter?  I think it is important to set it with a random number since it could generate different trees. Which could cause the code to crash.

Reply:
I was just wondering what random means here? Like 312459712347 is also a random number :-)
But it would not test the code very well. I believe that a low number will test our code the best, and also be somewhat slower. That's why we test with a lower number so we don't have to fill with as much data to get the same coverage. 
But also this is a parameter that always is fixed in core, so it's not important that it works with e.g. larger numbers. If it works with small numbers it very likely works in general.
So I think we can be more opinionated with numbers we want to test with, as testing with huge numbers will give worse code coverage/stress.
Do I make any sense ?

Reply:
Yes, it does make sense, let me rephrase the README, probably we should also remove it from the command line.

Reply:
@bmunkholm I decided to remove entirely the section about the NODE size, it is just confusing.